.nf-core.yml

```diff
@@ -15,9 +15,12 @@ lint:
     - .github/ISSUE_TEMPLATE/bug_report.yml
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
+    - .github/workflows/linting_comment.yml
```

Comment:
Could the `linting_comment.yml` and `linting.yml` files not be added to the files to ignore linting and instead the files themselves updated to match what's found e.g., in here: https://github.com/phac-nml/snvphylnfc/tree/dev/.github/workflows

Reply:
Thank you for the example! I will take care of that.

Reply:
Tests are passing locally, but no in actions. Do you have any suggestions on what could resolve the issue?